Makefile

```diff
@@ -248,9 +220,58 @@ rm -rf $$TMP_DIR ;\
 }
 endef
 
-#.PHONY: bundle
-#bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
-#	operator-sdk generate kustomize manifests -q
-#	cd config/manager && $(KUSTOMIZE) edit set image controller=$(TAG)
-#	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
-#	operator-sdk bundle validate ./bundle
+.PHONY: bundle
+bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
+	operator-sdk generate kustomize manifests -q
+	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
+	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	operator-sdk bundle validate ./bundle
+
+.PHONY: bundle-build
+bundle-build: ## Build the bundle image.
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
```

Comment:
should `${IMAGE_BUILDER}` be used here as well ?

Reply:
Yep, changed.